networkit/cpp/distance/BidirectionalBFS.cpp

```diff
@@ -15,7 +15,7 @@ void BidirectionalBFS::run() {
     if (G->isWeighted())
         WARN("Treating the graph as unweighted!");
 
-    stDist = 0;
+    distance = 0;
```

Comment:
If BidirectionalBFS is constructed using the new iterator-based target-list constructor, the algorithm would crash (due to `target=none`)?

Reply:
Good point, now both `BidirectionalBFS/Dijkstra` have their own constructor that takes a single target node.